Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:10.14.2
+# set a location
+WORKDIR /usr/pisa
+
+# also copy the build artifacts from the statechannels folder
+COPY ./statechannels/build ./statechannels/build
+
+# copy the package files
+COPY package*.json ./
+# install packages
+RUN ["npm", "ci"];
+# copy the src and the tsconfig
+COPY ./src ./src
+COPY ./tsconfig.json ./tsconfig.json
+# build
+RUN ["npm", "run", "build"]
+
+# expose the startup port
+EXPOSE 3000
+
+# start the application
+CMD ["npm", "run", "start-dev"]
```

Comment:
docker container should start production build instead of dev

Reply:
agreed. I couldnt get the normal command to work though, it had been holding me up for half a day so I decided to put this in a raise a ticket for the change. See #48

Reply:
cool